package.json

```diff
@@ -62,20 +67,10 @@
     "ethers": "^5.4.3",
     "ganache-core": "^2.13.2",
     "jest": "^27.0.6",
-    "jest-environment-ceramic": "^0.13.0",
+    "jest-environment-ceramic": "^0.14.0",
```

Comment:
Just a FYI, I updated this to remove the `lru_map` dependency. It seems that the older version of `jest-environment-ceramic` used a version of `@ceramicnetwork/core` that forgot to list `lru_map` as a dependency. So I'm guessing `lru_map` was set as a dependency as a work around.  I did not update the other dependencies as it is not in the scope of this ticket.